.github/workflows/ci.yml

```diff
@@ -26,13 +26,25 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
           node-version: 18
           cache: 'yarn'
 
-      - run: yarn install --frozen-lockfile --non-interactive
+      - name: Fetch yarn cache directory path
```

Comment:
why isn't this handled by setup-node's `cache: yarn`?

Reply:
@NullVoxPopuli it's too costly to build it in for GH as a default option. They cache Node.js version and npm, but not node_modules.